packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -899,7 +899,13 @@ abstract class SqlServerInstanceEngineBase extends InstanceEngineBase {
           majorVersion: props.version.sqlServerMajorVersion,
         }
         : undefined,
-      parameterGroupFamily: props.version ? `${props.engineType}-${props.version.sqlServerMajorVersion}` : undefined,
+      parameterGroupFamily: props.version
+        // for some reason, even though SQL Server major versions usually end in '.00',
```

Comment:
when it says *usually* how often is it something other than `.00`?
does this always need to be truncated to the first digit or does it have to end with `.0`?

Reply:
Currently, all major versions defined by constants end with `.00`. However, we also have the `of` methods that allow creating an arbitrary version (including a major version). I figured if there was a major version like `15.0`, we probably shouldn't change the parameter group family to be `sql-server-15.`.

Does this heuristic make sense?

Reply:
seems reasonable to me! thanks for clarifying